fmc_rest_client/resources/objects.py

```diff
@@ -91,3 +91,9 @@ def __init__(self, name=None, tag=None, desc=None):
         super().__init__(name)
         self.tag = tag
         self.description = desc
+
+class VariableSet(ObjectResource):
+    def __init__(self, name=None, desc=None):
+        super().__init__(name)
+        self.name = name
```

Comment:
name is already set in parent class constructor, you need not add name here again.